lib/cache.ts

```diff
@@ -0,0 +1,8 @@
+import { createStorage } from "unstorage";
+import fsDriver from "unstorage/drivers/fs";
+
+export const cacheBase = "node_modules/.cache/fontless/fonts/meta";
```

Comment:
I don't generally like hardcoding paths like this. It will probably work under windows, but some setups might not have a writeable node_modules directory.

Perhaps make this configurable via env ?

Reply:
Do you have an idea how we can handle this without it being configurable?

Reply:
@wmertens That's a good feedback, I'll merge the PR to going on with the POC implementations.
We will iterate on it later, Thanks